### ISSUE TITLE
Kata with accepted answer shows as not completed in search

### ISSUE THREAD

**Issue by Rashin46** [Codewars/codewars.com#1739]
I encountered with a kata where I have working (accepted) solution, but the site shows as not completed.

![image](https://user-images.githubusercontent.com/9058559/53287648-8ef05480-377f-11e9-9753-f0d94c31a3d4.png)

My solutions for this kata:

![53287665-cbbc4b80-377f-11e9-9f00-c33636830247](https://user-images.githubusercontent.com/9058559/53750403-f73ee480-3ea9-11e9-9edf-5e4363af2138.png)

Kata: https://www.codewars.com/kata/sort-and-star
User: Rashin46 (connected with this Github account)

Can you please check this issue?

Thank you in advance.

Best regards,
Rashin46

**Comment by cyril-lemaire** [comment]
Note: You might want to blur that screenshot as you're making a solution to this kata public (that doesn't matter much because it's an 8 kyu, but it's still bad practice)

**Comment by Rashin46** [comment]
Blurred the image.

**Comment by ivanduka** [comment]
Hi! I have exactly the same issue with kata https://www.codewars.com/kata/recursive-reverse-string
I completed successfully the kata but it still shows up in search as non-completed.
My username is IvandCodes (connected to this GitHub account).
Should I create a separate issue or just leave it here as a comment?

**Comment by FArekkusu** [comment]
I have the same issue. Solved [this kata](https://www.codewars.com/kata/57fafb6d2b5314c839000195) in Python, but it was still listed as not completed in kata search. Resubmitted my solution - no result. Solved it in JS - the kata is listed as completed now.

**Comment by kazk** [comment]
For some reason (maybe performance), a separate list of ids are used when searching and that seems to get outdated sometimes. I'm planning to get rid of it and find another way.

**Comment by sigod** [comment]
I had a similar issue with [Esolang Interpreters #1 - Introduction to Esolangs and My First Interpreter (MiniStringFuck)](https://www.codewars.com/kata/esolang-interpreters-number-1-introduction-to-esolangs-and-my-first-interpreter-ministringfuck). I wanted to check if I rated the kata, but it said the solutions are locked. So, I tried to "solve" the kata with the solution from my profile and it actually gave me more honor.

**Comment by hobovsky** [comment]
@sigod you encountered #1683, probably

**Comment by shiimao** [comment]
I have the same issue with the same symptoms as the original one (kata doesn't appear as completed even though I can vote on it and see my solution), but in addition I didn't receive any points nor progress. I did work on two different pcs with logging out and in before completing this kata - [Parsing and evaluation of mathematical expressions](https://www.codewars.com/kata/564d9ebde30917684f000048). I did try waiting 24 hours, completing katas to update stats, re-submitting my solution with no avail. I'm mildly dissapointed as this one required a bit of work specially for someone with no prior experience with C#. I know it's a part of broader issue but I see no harm in reporting it as well.
